.envrc.example

```diff
@@ -1,19 +1,9 @@
 toplevel="$(git rev-parse --show-toplevel)"
 
-# ensure the pre-commit git hook is in place
-pushd $toplevel/.git/hooks >/dev/null &&
-    ln -sf ../../.githooks/pre-commit &&
-    popd >/dev/null
+# build and use the managed go sdk
+unset GOROOT
+make -C "$toplevel" go-check
+go_bin_path="$(make --no-print-directory -C $toplevel go-bin-path)"
+PATH="$go_bin_path:$PATH"
```

Comment:
`make go-bin-path` already appends the rest of the PATH so suffixing with $PATH shouldn't be required

Reply:
Oh! No wonder the path looked so long.